lib/linters/url_quotes.js

```diff
@@ -8,45 +8,59 @@ module.exports = {
     message: 'URLs should be enclosed in quotes.',
 
     lint: function urlQuotesLinter (config, node) {
+        var results = [];
         var rSingle = /^\'(.+)\'$/;
         var rDouble = /^\"(.+)\"$/;
+        var self = this;
         var column;
         var value;
         var ast;
         var uri;
+        var func;
```

Comment:
Same thing as above, move it into the iteration perhaps? Just to contain it a little bit better.